### PR TITLE
Fix some formatting issues in translations

### PR DESCRIPTION
**Pull request by zonkmachine**
Many translations seem to have been done without verifying against a compiled version of lmms. Maybe on the Transifex online editor. Especially the instrument pluggin editor and the native instruments/effects has many short terms and acronyms that must stay short or the layout will go south. I've gone through all translation files and reverted the worst cases. It may be that my machine won't show all problematic translations so a second pair of eyes would be nice here.
Fixes https://github.com/LMMS/lmms/issues/4436
Fixes https://github.com/LMMS/lmms/issues/4285

PS. I've also fixed up some general issues in the Swedish translation.

### CODE REVIEW

File: data/locale/sv.ts [hunk 1]
```diff
@@ -1454,7 +1454,7 @@ Right clicking will bring up a context menu where you can change the order in wh
     <name>EnvelopeAndLfoView</name>
     <message>
         <source>DEL</source>
-        <translation>RADERA</translation>
```
Comment (zonkmachine) [inline data/locale/sv.ts]: :clown_face:

Comment (zonkmachine) [reply data/locale/sv.ts]: This is not only a horribly wrong translation but it fundamentally borks the envelope knob layout. People are busy using the Transifex online interface and don't double check against a compiled real version of lmms.

File: data/locale/sv.ts [hunk 2]
```diff
@@ -1063,7 +1063,7 @@ If you&apos;re interested in translating LMMS in another language or want to imp
     </message>
     <message>
         <source>FREQ</source>
-        <translation>FREKV.</translation>
+        <translation>FREK</translation>
```
Comment (zonkmachine) [inline data/locale/sv.ts]: Either this or just revert to no translation here.

File: data/locale/ja.ts [hunk 3]
```diff
@@ -9952,4 +9952,4 @@ The LED in the lower right corner of the waveform editor determines whether the
         <translation>出力ゲイン</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
```
Comment (zonkmachine) [inline data/locale/ja.ts]: One of the Japanese translators is working on a Windows machine...

File: data/locale/sv.ts (hunk 2 above)
Comment (zonkmachine) [reply data/locale/sv.ts]: I reverted those to Freq/FREQ and some other Instrument plugin knob texts to the English original for now.

**Comment by liushuyu** [comment]
Please upload the translation fix to Transifex otherwise they will be overriden in next translation update

**Comment by PhysSong** [comment]
> Please upload the translation fix to Transifex otherwise they will be overriden in next translation update

It requires some permission. @zonkmachine can update Swedish, and @liushuyu can update German.
There are no Japanese translators in our development team, but I guess @Umcaruje can push it because he's the [Team manager](https://docs.transifex.com/teams/understanding-user-roles#team-managers) for all languages.

**Comment by PhysSong** [comment]
Then @liushuyu, could you update translation on Transifex once this PR gets merged?

**Comment by liushuyu** [comment]
>Then @liushuyu, could you update translation on Transifex once this PR gets merged?

Sure thing.

File: data/locale/ja.ts (hunk 3 above)
Comment (zonkmachine) [reply data/locale/ja.ts]: I still don't get what's going on here. I haven't found a setting for formatting in my editor (xed). I think only the Swedish translation didn't see this and I haven't touched it until now I think. Are these format fixes correct or should I revert them?

**Comment by zonkmachine** [comment]
OK. I have reverted the translations that mess up formatting the most, especially in the Instrument Editor and the native pluggins, but have allowed some smaller issues as to keep this PR at a reasonable size.
This is now ready for review.

**Comment by zonkmachine** [comment]
Merge?

**Comment by zonkmachine** [comment]
For reviewing this I don't think there's any point in going through all of it. The changes are pretty much the same across all languages. Reverting to original English as the translated line is too long and has a negative effect on the knob layout. Just comparing a couple of the languages should be enough and then the translators can take it from there.

**Comment by zonkmachine** [comment]
I reverted some automatic formatting changes that my editor made. These edits may have been perfectly fine but I don't have time to look into it right now. Merging.

**Comment by PhysSong** [comment]
@liushuyu Could you please upload these changes to Transifex? It seems like you haven't.

**Pull request merged**